noxfile.py

```diff
@@ -11,7 +11,7 @@
 nox.options.sessions = "lint", "mypy", "pytype", "tests"
 
 
-@nox.session(python=["3.8", "3.7"])
+@nox.session(python=["3.9", "3.7"])
```

Comment:
Bruk nox_poetry.session, alt annet er utdatert i 0.8.1.

```suggestion
@nox_poetry.session(python=["3.9", "3.7"])
```